src/BenchmarkDotNet/Toolchains/MonoWasm/PInvokeTableGenerator.cs

```diff
@@ -0,0 +1,156 @@
+// -*- indent-tabs-mode: nil -*-
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
Not sure if BDN project is following the pattern the dotnet\runtime repo is, but they removed the third line from this versioning/header comments.

Reply:
I've already deleted this file; it was part of the WasmAppBuilder but turned out not to be needed.